pandas/core/series.py

```diff
@@ -490,13 +490,14 @@ def compress(self, condition, *args, **kwargs):
 
     def nonzero(self):
         """
-        Return the indices of the elements that are non-zero
+        Return the integer indices of the elements that are non-zero
```

Comment:
we normally call these indexers

Reply:
The confusion reflected in #19312 relates to the fact that the definitions are not clear. Personally, I don't know what are the different definitions. The `index` of a `Series` may be a default one; namely integers starting from `0`. Or, named indices. This PR stress in the documentation of `nonzero` that it returns the integers.